systest/testcontext/context.go

```diff
@@ -314,7 +314,7 @@ func New(t *testing.T, opts ...Opt) *Context {
 		Image:             imageFlag.Get(p),
 		PoetImage:         poetImage.Get(p),
 		NodeSelector:      nodeSelector.Get(p),
-		Log:               zaptest.NewLogger(t, zaptest.Level(logLevel)).Sugar(),
+		Log:               zaptest.NewLogger(t, zaptest.Level(logLevel)).Sugar().With("test", t.Name()),
```

Comment:
How about naming it instead:
```suggestion
		Log:               zaptest.NewLogger(t, zaptest.Level(logLevel)).Sugar().Named(t.Name()),
```